packages/material-ui/src/AccordionDetails/AccordionDetails.d.ts

```diff
@@ -12,7 +12,13 @@ export interface AccordionDetailsProps extends StandardProps<React.HTMLAttribute
   classes?: {
     /** Styles applied to the root element. */
     root?: string;
+    gutters?: string;
```

Comment:
We miss a description
```suggestion
    /** Styles applied to the root element unless `disableGutters={true}`. */
    gutters?: string;
```